documentation/modules/exploit/unix/webapp/joomla_sqli_rce_3_7_0.md

```diff
@@ -0,0 +1,49 @@
+## Vulnerable Application
+
+  This module exploits a SQL Injection vulnerability in the com_fields component which was introduced to the core of Joomla in version 3.7.0.
```

Comment:
i'd wrap com_fields in ticks (`) since its a component and will help it stand out visually